packages/hooks/README.md

```diff
@@ -0,0 +1,34 @@
+# WP-JS-Hooks
+
+A lightweight & efficient EventManager for JavaScript in WordPress.
+
+
+### API Usage
+API functions can be called via the global `wp.hooks` like this `wp.hooks.addAction()`, etc.
+
+* `addAction( 'namespace.identifier', callback, priority )`
+* `addFilter( 'namespace.identifier', callback, priority )`
+* `removeAction( 'namespace.identifier', callback )`
+* `removeFilter( 'namespace.identifier',  callback )`
+* `removeAllActions(  'namespace.identifier' )`
+* `removeAllFilters(  'namespace.identifier' )`
+* `doAction( 'namespace.identifier', arg1, arg2, moreArgs, finalArg )`
+* `applyFilters( 'namespace.identifier', content )`
+* `doingAction( 'namespace.identifier' )`
+* `doingFilter( 'namespace.identifier' )`
+* `didAction( 'namespace.identifier' )`
+* `didFilter( 'namespace.identifier' )`
+* `hasAction( 'namespace.identifier' )`
+* `hasFilter( 'namespace.identifier' )`
```

Comment:
As discussed during core JS office hours today, I'd like to see us alter how we refer to these components -- in jQuery and other libraries "namespace" refers to the your-code-specific part following the period, and the event name is the part before the period. I believe that breaking from that convention would lead to a lot of confusion.

What I would propose instead:
```js
addAction( 'actionname.namespace' );
```

Reply:
@kadamwhite this has been updated, can you plese review the current format?